plugins/node/opentelemetry-instrumentation-ioredis/src/ioredis.ts

```diff
@@ -81,9 +81,14 @@ export class IORedisInstrumentation extends InstrumentationBase<
   /**
    * Patch send command internal to trace requests
    */
-  private _patchSendCommand() {
+  private _patchSendCommand(moduleVersion?: string | undefined) {
```

Comment:
could be `moduleVersion?: string`

Reply:
Fixed, thanks.
Sorry for the delay, I only now seen this comment